osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -187,7 +229,27 @@ void genDetailsFromAddr(const struct ifaddrs* addr, QueryData& results) {
     r["odrops"] = INTEGER(0);
     r["collisions"] = BIGINT_FROM_UINT32(ifd->ifi_collisions);
     r["last_change"] = BIGINT_FROM_UINT32(ifd->ifi_lastchange.tv_sec);
-#endif
+    if (context.isColumnUsed("link_speed")) {
```

Comment:
Can you reorder the code so we can undent these large blocks?

Reply:
I started pulling this thread (assuming you mean reorder the #ifdef and if (context.isColumnUsed) ), and it involves shuffling a lot of the linux specific code.  It's a fine idea, but I'd prefer to do it in a separate change.